style/battle.css

```diff
@@ -630,7 +630,7 @@ License: GPLv2
 	background: #333333;
 	color: #CCCCCC;
 }
-.dark .battle-log .chat strong,
+.dark .battle-log .chat > strong,
```

Comment:
The really specific rules are my usual way? But you can also achieve this by adding CSS classes.